src/Apps/Order/Components/ExpressCheckout/ExpressCheckoutUI.tsx

```diff
@@ -524,19 +524,11 @@ const shippingRateForFulfillmentOption = option => {
   const { type, amount } = option
   switch (type) {
     case "DOMESTIC_FLAT":
-      if (amount) {
-        return {
-          id: type,
-          displayName: "Domestic shipping",
-          amount: amount.minor,
```

Comment:
Not sure if it was on purpose to have domestic without the `!`

Reply:
Yeah, also not sure if it's needed, I will refer to @rquartararo or @erikdstock